addons/survey/views/survey_templates_management.xml

```diff
@@ -73,6 +73,24 @@
         </t>
     </template>
 
+    <!-- Access error survey page -->
+    <template id="survey_access_error" name="Survey: access error">
+        <t t-call="survey.layout">
+            <div class="wrap">
+                <div class="container">
+                    <div class="jumbotron mt32">
+                        <h1>Survey Access Error</h1>
+                        <p>Oopsie! We could not let you open this survey. Make sure you are using the correct link and are allowed to
+                        participate or get in touch with its organizer.</p>
+                        <a class="btn btn-secondary" href="/web">
```

Comment:
Maybe we could redirect to the root (just "/") instead of the backend ?

It's more generic and should work for all use cases.